cps_stage3/extrapolation.py

```diff
@@ -203,7 +203,7 @@ def _read_data(self, growth_rates, cps_benefit, cps_weights,
         for benefit in benefit_names:
 
             # create benefit targets
-            benefit_2014 = (cps_benefit[benefit] * cps_benefit.s006).sum()
+            benefit_2014 = (cps_benefit[benefit] * cps_benefit.WT).sum()
```

Comment:
I'm not super familiar with the previous version of this script, so just want to check: is this a switch from s006 to WT for every year or just a renaming thing? In other words, are these two variables completely equivalent for every year?

Reply:
Yes. When we create the `s006` variable for the final CPS value we're just renaming the `WT` column. I'm actually still trying to figure out why it could run at all using `s006` instead of `WT`.